app/javascript/src/lib/dayjs.js

```diff
@@ -0,0 +1,6 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+dayjs.extend(relativeTime);
+
+export default dayjs;
```

Comment:
https://www.loom.com/share/4fb55c58719749eabce5bc095799b59c

- https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F

Reply:
Yedhin : https://www.loom.com/share/634a5a15c5774a2fb7c754438ab6d7aa

Reply:
I have created  a small loom video link in the above-mentioned comment 
I have implemented/written this logic as suggested in BigBinary academy article